src/main/java/lotto/exception/InvalidRangeLottoNumberException.java

```diff
@@ -0,0 +1,9 @@
+package lotto.exception;
+
+public class InvalidRangeLottoNumberException extends IllegalArgumentException {
+    private static final String ERROR_MESSAGE = "[ERROR] 로또 번호가 1 ~ 45 사이의 수가 아닙니다.";
```

Comment:
이런 부분들 역시 요구사항의 변경에 유연하게 대처하기 위해선 상수를 활용해 줄 수 있을 것 같아요.
```suggestion
    private static final String ERROR_MESSAGE = String.format("[ERROR] 로또 번호가 %d ~ %d 사이의 수가 아닙니다.", LottoConst.LOTTO_RANGE_START, LottoConst.LOTTO_RANGE_END);
```